packages/nux/src/components/inline-tip/test/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import TestRenderer from 'react-test-renderer';
+
+/**
+ * WordPress dependencies
+ */
+import { Notice } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { InlineTip } from '../';
+
+describe( 'InlineTip', () => {
+	it( 'should not render anything if invisible', () => {
+		const renderer = TestRenderer.create(
+			<InlineTip isVisible={ false }>
+				It looks like you’re writing a letter. Would you like help?
+			</InlineTip>
+		);
+		expect( renderer.root.children ).toHaveLength( 0 );
+	} );
+
+	it( 'should render correctly', () => {
+		const renderer = TestRenderer.create(
+			<InlineTip isVisible>
+				It looks like you’re writing a letter. Would you like help?
+			</InlineTip>
+		);
+		expect( renderer ).toMatchSnapshot();
+	} );
+
+	it( 'should call onRemove when the remove button is clicked', () => {
+		const onRemove = jest.fn();
+		const renderer = TestRenderer.create(
+			<InlineTip isVisible onRemove={ onRemove }>
+				It looks like you’re writing a letter. Would you like help?
+			</InlineTip>
+		);
+		renderer.root.findByType( Notice ).props.onRemove();
+		expect( onRemove ).toHaveBeenCalled();
```

Comment:
Feels like this should be an e2e test.

In any case, the description doesn't describe what the test actually does. Maybe something like 'calls through to the `InlineTip` `isRemove` prop when when the rendered `Notice` `onRemove` prop is called'.

Reply:
There are two E2E tests in `nux.test.js` that cover clicking the dismiss button. This is just a (useless?) unit test.

I've tweaked the name to be a little more accurate 🙂 964a75d3ab920ddefa7c3b6001002f5fd399876a